README.md

```diff
@@ -194,6 +194,58 @@ gamebot cancel
 Victor Barna and Deng Yaping canceled a challenge against Wang Hoe and Zhang Jike.
 ```
 
+#### gamebot automatch on|off
+
+Turn doubles automatch on for 5 minutes, or turn it off. When 4 players have automatch on, gamebot will create a new challenge with the players with the highest and lowest ELO scores vs the players with the middle scores.
+
+```
+gamebot automatch on
```

Comment:
This could be more fun, like `gamebot bored` and `gamebot busy` :) I guess `automatch` is fine, but think about a more fun language?